### PR TITLE
Updated RAV4 Hybrid TSS2 Port

### PR DESCRIPTION
**Pull request by illumiN8i**
# Car support
This pull requests adds support for Toyota - RAV4 Hybrid - 2019, 2020 - XSE, XLE, Limited.

This is an updated version of PR #740 

This is an explorer link to a drive with the **stock** system enabled on **Limited** trim from discord user bikamake: https://my.comma.ai/cabana/?route=37a1ee26e697b2f5%7C2020-01-02--17-02-35&exp=1609539781&sig=27Sw59xBx4gNpEZVjYJvJcCb1duIyL8q6HZwBtMn5r0%3D&max=8&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F37a1ee26e697b2f5%2F6cda466a54bfd3d23b993080d8d0e04e_2020-01-02--17-02-35

This is an explorer link to a drive with **openpilot** system enabled on **XLE** from discord user david (.ghost): https://my.comma.ai/cabana/?route=7e34a988419b5307%7C2019-12-18--19-13-30&exp=1608234361&sig=ijJbjI0Hohe5lXLsCMZdjhIkgC0ZWWejJFsHjTxmssI%3D&max=4&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F7e34a988419b5307%2F264bd7952ade031f4318a13b4de87227_2019-12-18--19-13-30

This is an explorer link to a drive with the **stock** system enabled on **XSE** trim from discord user sideplugmike:
Stock drive: https://my.comma.ai/cabana/?route=689f254b15632b32%7C2019-12-18--17-16-45&exp=1608309192&sig=ab%2FQM3baV9GMWf00M6Ov5kkPl%2BkzkR%2BnraMU14dANtM%3D&max=21&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F689f254b15632b32%2Ff1eb650f3f355dd869ee9e3c345373e5_2019-12-18--17-16-45

Stock drive from discord user RDuke to confirm the addition of 913:8 to support Swiss 2019 RAV4 Hybrid XLE: https://my.comma.ai/cabana/?route=192a598e34926b1e%7C2020-01-11--08-19-10&exp=1610294691&sig=245p84BFx9ym4yvcbPWaY8Pqnw9EtBKY4g9aNCHViUw%3D&max=2&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F192a598e34926b1e%2Fb39e89b8bef01a01d0de71122fb613ca_2020-01-11--08-19-10

### CODE REVIEW

File: selfdrive/car/toyota/interface.py [hunk 1]
```diff
@@ -163,14 +163,14 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
...
-    elif candidate == CAR.RAV4_TSS2:
+    elif candidate in [CAR.RAV4_TSS2, CAR.RAV4H_TSS2]:
       stop_and_go = True
       ret.safetyParam = 73
       ret.wheelbase = 2.68986
       ret.steerRatio = 14.3
       tire_stiffness_factor = 0.7933
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.6], [0.1]]
-      ret.mass = 3370. * CV.LB_TO_KG + STD_CARGO_KG
+      ret.mass = 3585. * CV.LB_TO_KG + STD_CARGO_KG # mean between normal and hybrid
```
Comment (geohot) [inline selfdrive/car/toyota/interface.py]: Huh, why do the mean here? Can't it be an if?

**Comment by geohot** [comment]
@pd0wm Do we have an automated check that the fingerprints don't conflict?

File: selfdrive/car/toyota/interface.py (hunk 1 above)
Comment (illumiN8i) [reply selfdrive/car/toyota/interface.py]: Good point. This was just copied from how other hybrids were done in the past. Fixed.

Comment (geohot) [reply selfdrive/car/toyota/interface.py]: Huh, yea, it's poorly done. This is good, or perhaps inline if if nothing else changes?

**Comment by pd0wm** [comment]
> @pd0wm Do we have an automated check that the fingerprints don't conflict?

Yes.

We just need to add it to the list of CI routes.

**Comment by geohot** [comment]
Approved once CI is back up and it's it there.

**Comment by pd0wm** [comment]
I'm working getting all our route sync scripts back up and running. Will update this PR once done.

**Comment by rmarcil1** [comment]
You. Can. Do. It!!!

**Comment by brianczapiga** [comment]
Now conflicts with (merged PR#965)

**Comment by brianczapiga** [comment]
Fingerprint now conflicts after merging #965

**Comment by pd0wm** [comment]
Uploaded this route to CI:

```python
  "7e34a988419b5307|2019-12-18--19-13-30": {
    'carFingerprint': TOYOTA.RAV4H_TSS2,
    'enableCamera': True,
  },
```

Can you add that to the list of test_car_models and rebase with master?

**Comment by pd0wm** [comment]
About the Isreali Corolla fingerprint, let's remove it in this PR so CI passes. I'll try to get FW query fingerprinting working asap, so then this should not be a problem anymore.

**Comment by illumiN8i** [comment]
Done

**Pull request merged**